test/Microsoft.AspNetCore.Hosting.FunctionalTests/WebHostBuilderTests.cs

```diff
@@ -43,7 +43,12 @@ public class WebHostBuilderTests : LoggedTest
                 {
                     TargetFramework = runtimeFlavor == RuntimeFlavor.Clr ? "net461" : "netcoreapp2.0",
                     ApplicationType = ApplicationType.Portable,
-                    StatusMessagesEnabled = false
+                    StatusMessagesEnabled = false,
+#if DEBUG
```

Comment:
@pranavkm @natemcmaster any ideas how we can avoid #if DEBUG in every test?  I found https://github.com/aspnet/BuildTools/blob/c390edb6f73e3ab992402835c004414c1fadfd4a/files/KoreBuild/KoreBuild.Common.props#L33 for how we enable Release builds for appveyor and travis, and the CI driectly sets Configuration via env var.